### DISCUSSION TITLE
Running a query in GraphiQL inside dev tools renders [object Object] when error is returned

### DISCUSSION THREAD

**Discussion by hinok** [apollographql/apollo-client-devtools#431] [Ideas]
Like on screenshot below

<img width="1552" alt="Zrzut ekranu 2020-06-1 o 14 17 46" src="https://user-images.githubusercontent.com/1313605/83408973-d00bf580-a413-11ea-9686-c27b3e6a430a.png">

Response from server 200 - OK

```json
[{
    "errors": [{
        "message": "401",
        "category": "server",
        "locations": [{
            "line": 2,
            "column": 3
        }],
        "path": ["..."],
        "trace": [],
    "data": {
        "result": null
    }
}]
```

**Comment by jcreighton** [comment]
@hinok Could let me know your version of Apollo Client, devtools version and browser version?

**Comment by hinok** [comment]
@jcreighton 

```
├── @apollo/react-hoc@3.1.3
├── @apollo/react-hooks@3.1.3
├── @apollo/react-ssr@3.1.3
├── apollo-cache@1.3.4
├── apollo-cache-inmemory@1.6.5
├── apollo-client@2.6.8
├── apollo-link@1.2.13
├── apollo-link-batch-http@1.2.13
├── apollo-link-error@1.1.12
├── apollo-upload-client@12.1.0
├── apollo-utilities@1.3.3
```

```
Chrome 83.0.4103.61
Apollo Client Developer Tools 2.3.0
```

**Comment by mattgabor** [comment]
I'm experiencing the same issue, `[Object Object]` for all errors:

```
"@apollo/client": "3.0.0-rc.12",
"@apollo/react-hooks": "3.1.3",
```

```
Chrome 83.0.4103.116 
Apollo Client Developer Tools 2.3.0
```

**Comment by tstreamDOTh** [comment]
Is this resolved? I am facing the same

**Comment by tstreamDOTh** [comment]
Okay, I got the issues - it's auth! 
The extension should allow the user to send headers as well. GraphQL server might be doing authentication based on headers set. 
If this feature doesn't exist I can volunteer to add and fix it

**Comment by duongntse** [comment]
Me too, I'm facing the same 

"@apollo/client": "^3.3.7",

Microsoft Edge Version 88.0.705.53
Apollo Client Developer Tools 2.3.5

**Comment by hwillson** [comment]
It sounds like the issue here is that we need to allow custom headers. I'll move this to a feature request. Thanks!
